src/InstrumentationEngine/dllmain.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "stdafx.h"
 
-#include "ProfilerManager.h"
+#include "ProfilerManagerWrapper.h"
```

Comment:
Why does dllmain.cpp need to include this?

Reply:
Artifact of past change, reverting

---
In reply to: [331295942](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r331295942) [](ancestors = 331295942)